poll/public/handlebars/poll_studio.handlebars

```diff
@@ -0,0 +1,22 @@
+<script id="poll-form-component" type="text/html">
+{{#each items}}
+<li class="field comp-setting-entry is-set poll-{{noun}}-studio-item">
+    <div class="wrapper-comp-setting">
+        <label class="label setting-label poll-setting-label" for="{{noun}}-label-{{key}}">{{noun}}</label>
+        <input class="input setting-input" name="{{noun}}-label-{{key}}" id="{{noun}}-label-{{key}}" value="{{text}}" type="text" /><br />
+        {{#if image}}
+        <label class="label setting-label" for="{{noun}}-img-{{key}}">Image URL</label>
+        <input class="input setting-input" name="{{noun}}-img-{{key}}" id="{{noun}}-img-{{key}}" value="{{img}}" type="text" />
```

Comment:
There needs to be a mechanism for describing the image here.  This is imperative.  A description of the image would be required to differentiate one choice from the other for users who cannot see.
http://www.w3.org/html/wg/drafts/html/CR/embedded-content-0.html#alt

I also recommend adding text that helps the author understand the meaning of useful alternatives (i.e. "Describe this image in a way that would allow someone to answer the poll if the image did not load")